Classes/Flowpack/ElasticSearch/ContentRepositoryAdaptor/Eel/ElasticSearchQueryBuilder.php

```diff
@@ -126,6 +126,11 @@ class ElasticSearchQueryBuilder implements QueryBuilderInterface, ProtectedConte
      */
     protected $result = array();
 
+    /**
+     * @Flow\InjectConfiguration(package="TYPO3.TYPO3CR.Search")
```

Comment:
@baschny: the highlighting feature/-configuration is kinda Elasticsearch specific, so I suggest using a separate configuration section with the current package name: "Flowpack.ElasticSearch.ContentRepositoryAdaptor" instead of using the existing "TYPO3.TYPO3CR.Search" conf. key.